src/doc/book/testing.md

```diff
@@ -499,6 +499,10 @@ be imported in every test with `mod common;`
 That's all there is to the `tests` directory. The `tests` module isn't needed
 here, since the whole thing is focused on tests.
 
+Note, when building integration tests, cargo will not pass the test flag to the
```

Comment:
@frewsxcv you want me to add a link to this page or add single quote around `test`?